lib/tesla/mock.ex

```diff
@@ -57,6 +57,38 @@ defmodule Tesla.Mock do
     %{method: :post} -> {201, %{}, %{id: 42}}
   end
   ```
+
+  ### Global mocks
+  By default, mocks are bound to the current process,
+  i.e. the process running a single test case.
+  This design allows proper isolation between test cases
+  and make testing in parallel (`async: true`) possible.
+
+  While this style is recommended, there is one drawback:
+  if Tesla client is called from different process then the
+  test case it will not get proper mock.
```

Comment:
I suggest a minor rewording to:
> If Tesla client is called from a different process then the test case will not be mocked.

Reply:
The test case process itself will be mocked. Either way, both messages would be wrong ;)

It should be something like this "if Tesla client is called from different process it will not use the setup mock"

Reply:
Looks good :-)